File: totosuki/ABC/B/161.py
N, M = map(int, input().split())
A = list(map(int, input().split()))
vote_sum = sum(A)
cnt = 0

for i in range(N):
  if A[i] >= vote_sum * (1 / (4*M)):
    cnt += 1

if cnt >= M:
  print("Yes")
else:
  print("No")

File: OCH_Tao/ABC/B/161.js
const input = require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n");
const [N, M] = input[0].split(" ").map(x => parseInt(x, 10));
const A = input[1].split(" ").map(x => parseInt(x, 10));
console.log(A.filter(x => x >= A.reduce((a, b) => a + b, 0) / (4 * M)).length >= M ? "Yes" : "No");